src/drtsans/reductionlog.py

```diff
@@ -301,12 +336,58 @@ def __save_individual_iq_to_log(iq=None, topEntry=None, entryNameExt=""):
         _create_groupe(entry=entry, name="Wavelength", data=wavelength, units="A")
```

Comment:
can  you rename this function to `_create_group`?
I think Jean sneaked in some French when he wrote this :smile: